### PR TITLE
167424646: setup heroku

### PR DESCRIPTION
**Pull request by araaliFarooq**
**What does this PR do?**
- setups and deploys the application to Heroku

**Description of Task to be completed?**

- setting up Heroku server
- connect the application to Heroku Git remote repository.

**How should this be manually tested?**

* test it in a browser by navigating to [this link](https://getnyumba.herokuapp.com/)

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

[ #167424646](https://www.pivotaltracker.com/story/show/167424646)

**Screenshots (if appropriate)**
Screen Shot 2019-07-23 at 11 31 21 AM

**Questions:**

- N/A

### CODE REVIEW

File: getnyumba/getnyumba/settings.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '$lk5(nj@w*h$gy&=+$#r^d@-zaf5k8!k!rc@sx7dm*n5qt-hm^'
```
Comment (Qadriyah) [inline getnyumba/getnyumba/settings.py:24]: This key should be put in the environment variables.

**Review by Qadriyah — CHANGES_REQUESTED** [review]

File: getnyumba/getnyumba/settings.py (hunk 1 above)
Comment (araaliFarooq) [reply getnyumba/getnyumba/settings.py:24]: yah, that makes more sense. Thanks.

**Comment by Qadriyah** [comment]
This PR merges to `master`. I would suggest the default branch be set to `develop` to avoid creating PRs that merge directly to `master`

**Comment by araaliFarooq** [comment]
> This PR merges to `master`. I would suggest the default branch be set to `develop` to avoid creating PRs that merge directly to `master`

Hehe.. I honestly thought we were merging to develop. Guess @BarnaTB forgot to configure that. Thanks for this. We shall see it sorted.

**Comment by MuhweziDeo** [comment]
what kind of database are we using

**Review by MuhweziDeo — APPROVED** [review]

File: getnyumba/getnyumba/settings.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = ['getnyumba.herokuapp.com']
```
Comment (BarnaTB) [inline getnyumba/getnyumba/settings.py:29]: Please change this value and put it in the environment variables as well. These allowed hosts could change as the application grows.

**Review by BarnaTB — CHANGES_REQUESTED** [review]

**Review by BarnaTB — CHANGES_REQUESTED** [review]
Also kindly squash your commits into one and change the base branch of this PR to `develop`. I have changed that configuration. Thanks.

**Pull request closed without merging**